feature-libs/user/account/components/verification-token-form/verification-token-dialog.component.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { FocusDirective, LaunchDialogService } from '@spartacus/storefront';
+import {
+  VERIFICATION_TOKEN_DIALOG_ACTION,
+  VerificationTokenDialogComponent,
+} from './verification-token-dialog.component';
+
+@Pipe({
+  name: 'cxTranslate',
+})
+class MockTranslatePipe implements PipeTransform {
+  transform(): any {}
+}
+
+class MockLaunchDialogService implements Partial<LaunchDialogService> {
+  closeDialog(_reason: any) {}
+}
+
+describe('AsmBindCartDialogComponent', () => {
```

Comment:
I think it should be `VerificationTokenDialogComponent` instead of  `AsmBindCartDialogComponent`.

Reply:
fixed